src/algebra/big_operators/finprod.lean

```diff
@@ -628,6 +628,16 @@ begin
   exact finprod_mem_congr rfl he₁
 end
 
+/-- The product of `f i` is equal to the product of `g j` if there exists a bijective function
+`e : α → β` such that for all `x` we have `f x = g (e x)`. -/
+@[to_additive] lemma finprod_eq_of_bijective {f : α → M} {g : β → M}
```

Comment:
Is there a finset version of this lemma? If so, can you reference it in the docstring?

Reply:
There is `fintype.prod_bijective`, I'll and references both ways and maybe some more lemmas then to match the fintype api

Reply:
Oh and there is `finset.prod_bij` which is maybe what you meant?

Reply:
`fintype.prod_bijective` was the one I was thinking of